metricflow/configuration/yaml_handler.py

```diff
@@ -30,6 +30,13 @@ def get_value(self, key: str) -> str:
             raise ValueError(f"Value for key '{key}' cannot be None in the yaml file '{self.yaml_file_path}")
         return value
 
+    def get_value_safe(self, key: str) -> Optional[str]:
+        """Attempts to get a value from the yaml file, but return None instead of throwing an exception."""
+        try:
+            return self.get_value(key)
+        except Exception:
```

Comment:
https://github.com/transform-data/metricflow/blob/8d590b56db1b71ece2fc6884091ca291ec973d75/metricflow/configuration/yaml_handler.py#L21 you can see the 2 possible Exceptions thrown in the `get_value` function

Reply:
Having a broad exception handler like this is not great as it would also mask an exception caused by a coding error. How about putting in a similar implementation to `get_value` but returning None when it would throw an exception?

Reply:
how bout something like this 
```
try:
    self._email = os.getenv(TelemetryReporter.ENV_EMAIL_OVERRIDE) or ConfigHandler().get_value(CONFIG_EMAIL)
except Exception as e:
    logger.warn(f"ERROR retrieving email: {str(e)}")
    self._email = None
```